responses.go

```diff
@@ -371,6 +372,11 @@ type DataAccess struct {
 	Sequence int         `json:"sequence"`
 }
 
+type ReturnValueData struct {
+	Id      Int64  `json:"first"`
+	Message string `json:"second"`
```

Comment:
Is it really `first` and `second` here and no `id` and `message`? Something seems weird here

Reply:
@maoueh I think you are right, we should decline this PR, I will do some more investigation and come with a better code proposal

Reply:
I reverted commit bf9faf5db9ddd91268693e500351992354baf63e.

The return value will be interesting to properly decode because knowledge of what the final type will be is required to make an appropriate unmarshalling.

If the `hex` version is always available `return_value_hex_data`, the Go code could decode it to the right object:

```
type Trace struct {
    ...
    ReturnValueHexData HexBytes `json:"return_value_hex_data,omitempty"`
    ...
}

func (t *Trace) DecodeReturnValue(v interface{}) error {
  // With some sanity checks here like is not empty
  buffer := bytes.NewBuffer(t.return_value_hex_data)
  decoder := NewDecoder(buf)
  
  return decoder.Decode(v)
}
```